pepper-apis/config/testing-circleci.conf.ctmpl

```diff
@@ -0,0 +1,183 @@
+# Use this file to run local tests using a disposable in-memory database
+# while connecting to other environment-specific resources
+
+{{with $environment := env "ENVIRONMENT"}}
+{{with $version := env "VERSION"}}
+{{with $testingConf := vault (printf "secret/pepper/%s/%s/testing" $environment $version)}}
+{{with $auth0 := index $testingConf.Data "auth0"}}
+{{with $testingAuth := index $testingConf.Data "auth0"}}
+{{with $elasticsearchConf := vault (printf "secret/pepper/%s/%s/elasticsearch" $environment $version)}}
+
+{
+    # If set to true, tests will use MySQL from Docker container and dbUrl and housekeepingDbUrl will be ignored
+    "useDisposableTestDbs": false,
+    "dbUrl": "jdbc:mysql://localhost:3306/studyservicedb?user=root&password=rootpw&characterEncoding=UTF-8&useLegacyDatetimeCode=false&serverTimezone=UTC&useSSL=false&sessionVariables=innodb_strict_mode=on,tx_isolation='READ-COMMITTED',sql_mode='TRADITIONAL'",
+    "housekeepingDbUrl": "jdbc:mysql://localhost:3306/housekeepingdb?user=root&password=rootpw&characterEncoding=UTF-8&useLegacyDatetimeCode=false&serverTimezone=UTC&useSSL=false&sessionVariables=innodb_strict_mode=on,tx_isolation='READ-COMMITTED',sql_mode='TRADITIONAL'",
+    "maxConnections": 50,
```

Comment:
Maybe we should look at consolidating configuration? Here are the main differences to run tests in CircleCi.: we start mysql server with a RAM disk (TestContainer was difficult to run in CircleCi) and needed to tweak the URLs to use this MySQL instance.

Reply:
I'll take a look at that. Just found an old error on the config file and had to correct in both. So this setup is not really acceptable.

Reply:
Consolidated testing config files: a670d35